src/js/components/useToast.test.ts

```diff
@@ -36,93 +36,88 @@ describe('useToast', () => {
     });
 
     it('should display the HTML markup message when used', () => {
-      const toast = useToast(Toastify.TestMarkupMessage);
-      toast.instance.show();
-
-      const toastMessage = toast.content.innerHTML;
+      const {instance, element, content} = useToast(Toastify.TestMarkupMessage);
+      instance?.show();
```

Comment:
```suggestion
      const {instance, element, content, show: showToast} = useToast(Toastify.TestMarkupMessage);
      showToast();
```

Use every methods like this, so you don't need to use the optional chaining